LibreNMS/Util/Html.php

```diff
@@ -185,4 +186,18 @@ public static function powerStateLabel($state): array
                 return ['UNKNOWN', 'label-default'];
         }
     }
+
+    public static function severityToLabel(Severity $severity, string $text): string
```

Comment:
You can add functions to enum classes. Might be a better central location. Not sure ?

Reply:
I didn't want to add html into the enum class.